resolwe_bio/descriptors/sample_descriptorschema.yaml

```diff
@@ -0,0 +1,344 @@
+- model: flow.descriptorschema
```

Comment:
Remove this file. No need for a fixture and a schema. Both are the same entry in the DB, just import is different.